cdc/puller/puller.go

```diff
@@ -36,8 +36,8 @@ import (
 )
 
 const (
-	defaultPullerEventChanSize  = 128000
-	defaultPullerOutputChanSize = 128000
+	defaultPullerEventChanSize  = 16
```

Comment:
Is this too small, kv client could receive batch events, while events sent to puller are one by one